saleor/webhook/event_types.py

```diff
@@ -86,6 +86,11 @@ class WebhookEventAsyncType:
     SALE_DELETED = "sale_deleted"
     SALE_TOGGLE = "sale_toggle"
 
+    PROMOTION_CREATED = "promotion_created"
+    PROMOTION_UPDATED = "promotion_updated"
+    PROMOTION_DELETED = "promotion_deleted"
+    PROMOTION_TOGGLE = "promotion_toggle"
+
```

Comment:
You should also define the new events in the `PERMISSIONS` section and match them with the expected permission.